persistence-modules/mybatis-vs-hibernate/src/test/java/com/baeldung/mybatisvshibernate/MybatisUnitTest.java

```diff
@@ -0,0 +1,52 @@
+package com.baeldung.mybatisvshibernate;
+
+import org.apache.ibatis.io.Resources;
+import org.apache.ibatis.jdbc.ScriptRunner;
+import org.apache.ibatis.session.SqlSession;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.ibatis.session.SqlSessionFactoryBuilder;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+
+
+class MybatisUnitTest {
+    static SqlSessionFactory ssf;
+    @BeforeAll
+    public static void initialize() throws IOException {
+        String resource = "mybatis-config.xml";
+        InputStream inputStream = Resources.getResourceAsStream(resource);
+        ssf = new SqlSessionFactoryBuilder().build(inputStream);
+        myBatis3Setup();
+    }
+    private static void myBatis3Setup() throws IOException {
+        try(SqlSession sf = ssf.openSession()) {
```

Comment:
This whitespace issue is a sign that you might not be using the Baeldung formatter? Can you apply it to all files so we have our standard formatting rules applied
```suggestion
        try (SqlSession sf = ssf.openSession()) {
```